packages/@expo/fingerprint/src/sourcer/Expo.ts

```diff
@@ -15,20 +15,42 @@ export async function getExpoConfigSourcesAsync(
   projectRoot: string,
   options: NormalizedOptions
 ): Promise<HashSource[]> {
-  const results: HashSource[] = [];
+  if (!resolveFrom.silent(path.resolve(projectRoot), 'expo/config')) {
+    return [];
+  }
 
+  const results: HashSource[] = [];
   let config: ProjectConfig;
+  let loadedModules: string[] = [];
+  const ignoredFile = await createTempIgnoredFileAsync(options);
   try {
-    const { getConfig } = require(resolveFrom(path.resolve(projectRoot), 'expo/config'));
-    config = await getConfig(projectRoot, { skipSDKVersionRequirement: true });
+    // TypeScript is only for unit tests
+    const isTypeScript = path.extname(__filename) === '.ts';
+    const nodeProgram = isTypeScript ? 'ts-node' : 'node';
+    const expoConfigLoaderPath = 'ExpoConfigLoader' + (isTypeScript ? '.ts' : '.js');
+    const { stdout } = await spawnAsync(
+      'npx',
```

Comment:
Not entirely sure but you may need to add a shebang to the file if you use npx(?)

Reply:
changed my mind to add a shebang because that would probably lose the support on windows. since we don't need ts-node execution for testing, we could just replace the `npx` here to `node`.